frameworks/elastic/versions.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
 
-export TEMPLATE_ELASTIC_VERSION="6.1.3"
-export TEMPLATE_ELASTIC_STATSD_VERSION="${TEMPLATE_ELASTIC_VERSION}.0"
-export TEMPLATE_SUPPORT_DIAGNOSTICS_VERSION="6.3"
+export TEMPLATE_ELASTIC_VERSION="5.6.5"
+export TEMPLATE_SUPPORT_DIAGNOSTICS_VERSION="6.2"
```

Comment:
We still need the `TEMPLATE_ELASTIC_STATSD_VERSION` setting here.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2274/commits/d008a4c985cddc0d75834994fa9a0b7eed0560b2